### PR TITLE
Login register

### PR DESCRIPTION
**Pull request by karissa**
begin on #328

### CODE REVIEW

File: client/js/components/auth/login.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const html = require('choo/html')
+const form = require('get-form-data')
+
+const login = (state, prev, send) => {
+  function onSubmit (e) {
```
Comment (juliangruber) [inline client/js/components/auth/login.js]: just to be consistent with the style, this can be a `const` as well

Comment (juliangruber) [inline client/js/components/auth/login.js]: (an the `data` var below)

Comment (juliangruber) [inline client/js/components/auth/login.js]: yeah JS is a mess right now, that's why it's even more important to work on code style

File: client/js/components/auth/login.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    var data = form(e.target)
+    send('user:login', data)
+    e.preventDefault()
+    return false
```
Comment (juliangruber) [inline client/js/components/auth/login.js]: the `return false` afaik is only necessary for older IEs, in which our tech stack wouldn't work anyways, right?

File: client/js/components/auth/login.js (hunk 1 above)
Comment (karissa) [inline client/js/components/auth/login.js]: ok thanks!

File: client/js/components/auth/login.js (hunk 2 above)
Comment (karissa) [inline client/js/components/auth/login.js]: removed it

**Comment by karissa** [comment]
@juliangruber this branch isn't ready yet i'll need another review later after I add more

**Comment by karissa** [comment]
@Kriesse current state. I'm going to merge this and then feel free to go to town! 
https://www.dropbox.com/s/m3geiwurto9ofoq/ready.mov?dl=0

**Comment by karissa** [comment]
@joehand now the auth api is at /api/v1/login and /api/v1/register

**Comment by joehand** [comment]
Sweet! Got the datfolder server working and passing auth tests in [dat-next](https://github.com/joehand/dat-next/pull/36)

**Pull request merged**